.github/workflows/kind-e2e.yaml

```diff
@@ -75,12 +75,15 @@ jobs:
         - k8s-version: v1.23.13
           kind-version: v0.17.0
           kind-image-sha: sha256:ef453bb7c79f0e3caba88d2067d4196f427794086a7d0df8df4f019d5e336b61
+          DAPR_TEST_CONFIG_STORE: "postgres"
         - k8s-version: v1.24.7
           kind-version: v0.17.0
           kind-image-sha: sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
+          DAPR_TEST_CONFIG_STORE: "redis"
         - k8s-version: v1.25.3
           kind-version: v0.17.0
           kind-image-sha: sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
+          DAPR_TEST_CONFIG_STORE: "redis"
```

Comment:
nit: please, keep the same naming convention. No all caps here.

Reply:
Done